configs/tasks/rearrange/pick_state.yaml

```diff
@@ -64,111 +70,93 @@ TASK:
         TYPE: "EndEffectorToObjectDistance"
     END_EFFECTOR_TO_REST_DISTANCE:
         TYPE: "EndEffectorToRestDistance"
-    REARRANGE_PICK_REWARD:
+    FORCE_TERMINATE:
+        TYPE: "ForceTerminate"
+        MAX_ACCUM_FORCE: 0.0
+    PICK_REWARD:
         TYPE: "RearrangePickReward"
-        DIST_REWARD: 20.0
-        SUCC_REWARD: 10.0
-        PICK_REWARD: 20.0
+        DIST_REWARD: 10.0
+        PICK_REWARD: 5.0
         CONSTRAINT_VIOLATE_PEN: 10.0
         DROP_PEN: 5.0
         WRONG_PICK_PEN: 5.0
-        COLL_PEN: 1.0
+        COLL_PEN: 0.0
```

Comment:
COLL_PEN is an abbreviation that is not immediately obvious to me. What are the conventions for naming these configurations? Is there a glossary somewhere?

Reply:
There is no glossary. I think it's important we get documentation for the configs but I am not sure where to put it.

Reply:
I updated the name.